pipelines/tekton/aiedge-e2e/aiedge-e2e.pipeline.yaml

```diff
@@ -352,17 +349,16 @@ spec:
       value: $(tasks.build-container.results.IMAGE_DIGEST)
     - name: pipeline-run-uid
       value: $(context.pipelineRun.uid)
-    - name: target-registry-url
-      value: quay.io/$(params.target-imagerepo)/$(params.model-name)
+    - name: target-image-tag-references
+      value: ["$(params.target-image-tag-references[*])"]
     runAfter:
     - test-model-rest-svc
     - stop-deployment
+    workspaces:
+    - name: images-url
+      workspace: buildah-cache
```

Comment:
Maybe this should be a different workspace?

Reply:
~@devguyio can you explain what you mean a little more here? What I'm thinking is that it's already being used as more than a buildah-cache, and especially this PR is adding something that's not related to the image build.~ The `skopeo-copy` explicitly needs a file called `url.txt` at the root of the workspace, and I guess at this stage in the pipeline, we shouldn't need to care if something had already put a file with the same name there, so I'll just put a `rm -f url.txt` in the script before appending to it (ugh, actually I'll need to do that anyway, because if the workspace is a reused PV across runs, then we'd be appending to it continuously, and pushing over existing tags...). 

Actually, I think we _at least_ need to rename this workspace from `buildah-cache` to something else, since _buildah isn't caching in this pipeline!_ (in the version (0.6) of the Task that we get from OpenShift Pipelines currently, there's a hardcoded `--no-cache` (the `--no-cache is removed in version 0.7, but that's to allow remote/distributed caching with `--cache-from`/`--cache-to` which that doesn't happen by default, and we don't do it, and it wouldn't be using this workspace to do the caching anyway even if we did).

I think maybe I'll just rename it to "build-workspace", LMK if you think it should be something different.

Reply:
yeah I'm fine with the name